components/post_view/post_list/post_list.jsx

```diff
@@ -265,6 +265,11 @@ export default class PostList extends React.PureComponent {
             return;
         }
 
+        // Initialize counter if undefined
+        if (!this.extraPagesLoaded) {
+            this.extraPagesLoaded = 0;
```

Comment:
Could we actually set this in the constructor? I don't think we need to wait until here to fix the type, and that'll make our lives easier when we convert this to TypeScript (which is being done in #10624)

Reply:
> convert this to TypeScript

That's awesome! 🔥 

> set this in the constructor

Done!